EventFilter/TotemRawToDigi/test/analyze_vfat_frames.py

```diff
@@ -0,0 +1,43 @@
+import FWCore.ParameterSet.Config as cms
+import FWCore.ParameterSet.VarParsing as VarParsing
+
+process = cms.Process("AnalyzeVFATFrames")
+
+# default options
+options = VarParsing.VarParsing ('analysis')
+options.inputFiles= 'file:/afs/cern.ch/user/j/jkaspar/public/run268608_ls0001_streamA_StorageManager.root',
```

Comment:
we advise against personal files exposed in the release area.
Please see if something in /store/ (can even be /store/user or /store/group) can be used instead.